src/adapters/TelnetAdapter.ts

```diff
@@ -1066,7 +1070,7 @@ export class TelnetAdapter {
         if (this.requestPipeline) {
             let commandsExecuted = 0;
             do {
-                let data = await this.requestPipeline.executeCommand(`exit`, false);
```

Comment:
thought we removed this?

Reply:
We removed the call, but I hadn't removed the actual function. Just verified, it's not called anywhere in the project. 

![image](https://user-images.githubusercontent.com/2544493/145224697-d8ae3beb-cedf-46ca-992d-13b3cd705e25.png)

Reply:
👍  ok sweet. We should remove it then. Can be it's ow PR or not. Up to you.

Reply:
Will do it separately, so we have a commit to track it.